src/libs/actions/PersonalDetails.ts

```diff
@@ -157,6 +158,21 @@ function updateDateOfBirth({dob}: DateOfBirthForm) {
     Navigation.goBack();
 }
 
+function updatePhoneNumber(phoneNumber: string) {
```

Comment:
@allgandalf we need to handle error here

Reply:
@mountiny Wdyt?

Reply:
We should add rbr in case of an error from api that will show error message under the input